src/components/modal/Modal.tsx

```diff
@@ -16,41 +16,59 @@ interface ModalOwnProps {
   closeTimeout?: number;
   shouldCloseOnOverlayClick?: boolean;
   customClassName?: string;
+  overlayClassName?: string;
   children?: React.ReactNode;
+  shouldCloseOnEsc?: boolean;
+  bodyOpenClassName?: string;
+  portalClassName?: string;
 }
 
 ReactModal.defaultStyles = {};
 ReactModal.setAppElement("#root");
 
-function Modal(props: ModalOwnProps) {
-  const {
-    isOpen,
-    customClassName,
-    children,
-    closeTimeout = MODAL_CLOSE_TIMEOUT,
-    canBeClosed = true,
-    modalContentLabel,
-    onAfterClose,
-    onAfterOpen
-  } = props;
+function getParent() {
+  return document.querySelector("#modal-root") as HTMLElement;
+}
+
+function Modal({
+  isOpen,
+  customClassName,
+  children,
+  closeTimeout = MODAL_CLOSE_TIMEOUT,
+  canBeClosed = true,
```

Comment:
What was the purpose of this `canBeClosed`? In which scenario we may pass `canBeClosed={false}`?

Reply:
Here is an example use case from Pascal:

User creates an article. After the article is successfully created, we display a modal asking the user whether they want to go ahead and distribute the article, or just wants to navigate to article detail without distributing it.

In this case, the modal shouldn't be "closable" (not sure if that's a word).

Reply:
It could probably be done by providing these two props to the Modal, instead of having another prop like `canBeClosed`: `shouldCloseOnOverlayClick` and `shouldCloseOnEsc`.

Feel free to open a PR or create an issue about this.

Reply:
Thanks for the explanation 🙏  
I created an issue to rename the prop as `canBeClosedWithoutAction`. I can handle it after this PR is merged. See the issue here: #17